scripts/run-integration-tests.sh

```diff
@@ -35,6 +35,7 @@ __cluster_deprovisioned=0
 
 on_error() {
     echo "Error with exit code $1 occurred on line $2"
+    emit_cloudwatch_metric "error_occurred" "1"
```

Comment:
Here we won't know which test failed right? or do we get that granularity in CW?

Reply:
Synced up with Vikas on this.